src/transport/AdminPairingTable.h

```diff
@@ -123,6 +129,7 @@ class DLL_EXPORT AdminPairingInfo
         uint64_t mNodeId;   /* This field is serialized in LittleEndian byte order */
         uint64_t mFabricId; /* This field is serialized in LittleEndian byte order */
         uint16_t mVendorId; /* This field is serialized in LittleEndian byte order */
+        char mFabricLabel[32]; 
```

Comment:
Leave space for null terminator here as well.

Reply:
left space for the first byte specifying length.